src/Tasks/LegalProposition/Quiz.hs

```diff
@@ -21,7 +22,6 @@ import Trees.Types (BinOp, SynTree)
 
 
 
-
```

Comment:
Please avoid such diffs.

Reply:
I usually make these changes subconsciously. I will try my best to filter out those changes.